use-cases/predictive_maintenance/1_dataprep_dw_job_predmaint.ipynb

```diff
@@ -4,18 +4,45 @@
    "cell_type": "markdown",
```

Comment:
<p>Link text <code>XGBoost Algorithm</code> instead of raw link. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done</p><p> </p><p> </p>